### PR TITLE
Updated readme with contribution guidelines [Issue #79]

### PR DESCRIPTION
**Pull request by jusleg**
Fixes: #79 
### WHAT kind of change does this PR introduce?
Update to the readme to include contributing guidelines.

### HOW is this accomplished?
Updating the readme.

### Checklist
- [x] Docs have been added / updated to reflect the changes
- [x] I have reviewed and tested the changes :white_check_mark:
- [x] I have squashed my commits to have a reasonable amount of commits
- [x] All of the code I have written adhere to the coding conventions outlined in the coding conventions section of the wiki

### CODE REVIEW

**Comment by almiche** [comment]
✅| Styling is adequate and follows coding conventions outlined by the [Kotlin Foundation](https://kotlinlang.org/docs/reference/coding-conventions.html)

N/A

✅| New methods and functionalities have appropriate tests (testing edge cases + good coverage)

N/A

✅| Branch has an acceptable amount of commits with descriptive names and

N/A

✅| PR description matches the code outlined a has mentions to issues

N/A

✅| This PR attempts to implement a specific feature (no code blob or epic pr)

N/A

Status: **Approved**
Comments:

File: README.md [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
-
 <img alt="App image" src="screenshots/app.png" width="250">
 <img alt="App image" src="screenshots/app_2.png" width="250">
-<img alt="App image" src="screenshots/widget.png" width="250">
 
-It contains a couple UI and unit tests, they can be ran with the following instructions.
+## Contributing
+
+To submit new changes to the codebase, a pull request must be opened. The pull request must have the following:
+  1. Name that describes the modification
```
Comment (0matleb2) [inline README.md]: and includes reference(s) to issue(s)

Comment (jusleg) [reply README.md]: done
